pkg/scheduler/framework/preemption/preemption_test.go

```diff
@@ -279,8 +279,10 @@ func TestDryRunPreemption(t *testing.T) {
 			}
 			informerFactory := informers.NewSharedInformerFactory(clientsetfake.NewSimpleClientset(objs...), 0)
 			parallelism := parallelize.DefaultParallelism
+			ctx, cancel := context.WithCancel(context.Background())
```

Comment:
use this `ctx in L316 `pe.DryRunPreemption(context.Background(), pod, nodeInfos, nil, 0, int32(len(nodeInfos)))`